vmdb/lib/miq_automation_engine/service_models/miq_ae_service_cloud_volume.rb

```diff
@@ -0,0 +1,10 @@
+module MiqAeMethodService
+  class MiqAeServiceCloudVolume < MiqAeServiceModelBase
+    expose :ext_management_system,  :association => true
+    expose :availability_zone,      :association => true
+    expose :cloud_tenant,           :association => true
+    expose :base_snapshot,          :association => true
+    expose :cloud_volume_snapshots, :association => true
+    expose :attachments,            :association => true
```

Comment:
The attachments association returns a ```Disk``` model which is not defined as a service model yet so that also needs to be created.